indigo_app/templates/indigo_api/_task_detail.html

```diff
@@ -67,6 +67,28 @@ <h5>
         <div class="card">
           <div class="card-body">
             {{ task.description|default:''|urlize|linebreaksbr }}
+            {% if task.input_file %}
+              <div class="mt-2">
+                <h6>{% trans "Input" %}</h6>
```

Comment:
```suggestion
                <h6>{% trans "Input file" %}</h6>
```

Reply:
since we're moving our forms to "key: value" with `<dl>`, maybe use that here too? It makes it a bit clearer that it's not part of the description text, too.